packages/wallets/src/components/WalletsCarousel/WalletsCarousel.tsx

```diff
@@ -2,6 +2,10 @@ import React, { useState } from 'react';
 import useEmblaCarousel from 'embla-carousel-react';
 import { useWalletAccountsList } from '@deriv/api';
 import AccountsList from '../AccountsList';
+import { ProgressBar } from '../ProgressBar';
+import { WalletCard } from '../WalletCard';
+import { WalletListCardIActions } from '../WalletListCardIActions';
+import './WalletsCarousel.scss';
 
 const WalletsCarousel = () => {
     const [emblaRef, emblaApi] = useEmblaCarousel({ skipSnaps: true, containScroll: false });
```

Comment:
Please rename `emblaRef` to a more unique name otherwise it will not work when there's more than one carousel in page.